stdlib/lazy.mli

```diff
@@ -67,36 +66,76 @@ external force : 'a t -> 'a = "%lazy_force"
    recursively.
 *)
 
-val force_val : 'a t -> 'a
-(** [force_val x] forces the suspension [x] and returns its
-    result.  If [x] has already been forced, [force_val x]
-    returns the same value again without recomputing it.
 
-    If the computation of [x] raises an exception, it is unspecified
-    whether [force_val x] raises the same exception or {!Undefined}.
-    @raise Undefined if the forcing of [x] tries to force [x] itself
-    recursively.
+(** {1 Iterators} **)
+
+val map : ('a -> 'b) -> 'a t -> 'b t
+(** [map f x] returns a suspension that, when forced,
+    forces [x] and applies [f] to its value.
+
+    It is equivalent to [lazy (f (Lazy.force x))].
+
+    @since 4.13.0
 *)
 
+(** {1 Reasoning on already-forced suspensions} *)
```

Comment:
I'm not super fond of the title (I don't see how "reasoning" applies to `from_val` and `map_val`).  Perhaps just "{1 Already-forced suspensions}"?

Reply:
done, thanks